lib/active_interactor/organizer/callbacks.rb

```diff
@@ -141,11 +141,127 @@ def around_each_perform(*filters, &block)
         def before_each_perform(*filters, &block)
           set_callback(:each_perform, :before, *filters, &block)
         end
+
+        # Define a callback to call after all {Organizer::Organize::ClassMethods#organized organized}
+        # {ActiveInteractor::Base interactors'} {Interactor::Perform#perform #perform} methods have been called.
+        #
+        # @example
+        #  class MyInteractor1 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor1'
+        #    end
+        #  end
+        #
+        #  class MyInteractor2 < ActiveInteractor::Base
+        #    def perform
+        #      puts 'MyInteractor2'
+        #    end
+        #  end
+        #
+        #  class MyOrganizer < ActiveInteractor::Organizer
+        #    after_all_perform :print_done
+        #
+        #    organized MyInteractor1, MyInteractor2
+        #
+        #    private
+        #
+        #    def print_done
+        #      puts 'Done'
+        #    end
+        #  end
+        #
+        #  MyOrganizer.perform
+        #  "MyInteractor1"
+        #  "MyInteractor2"
+        #  "Done"
+        #  #=> <MyOrganizer::Context>
+        def after_all_perform(*filters, &block)
```

Comment:
Do you think it makes sense to define these examples/methods in this order:

1) Before
2) After
3) Around

Instead of:

1) After
2) Around
3) Before

Reply:
I'm following how the maintainer ordered the existing ones.